HWFlask-2.py

```diff
@@ -0,0 +1,38 @@
+from flask import Flask, render_template, request, redirect, make_response
+
+app = Flask(__name__)
+
+
+@app.route("/")
+def index():
+    return render_template("index.html")
+
+
+@app.route("/login", methods=["POST"])
+def login():
+    name = request.form.get("name")
+    email = request.form.get("email")
+
+    resp = make_response(redirect("/hello"))
+    resp.set_cookie("name", name)
+    resp.set_cookie("email", email)
+    return resp
+
+
+@app.route("/hello")
+def hello():
+    name = request.cookies.get("name")
+    email = request.cookies.get("email")
+    return render_template("hello.html", name=name, email=email)
+
+
+@app.route("/logout")
+def logout():
+    response = make_response(redirect("/"))
+    response.delete_cookie("name")
+    response.delete_cookie("email")
+    return response
```

Comment:
Функций представления может быть всего 2
```suggestion
@app.route("/", methods=['GET',"POST"])
def index():
    if request.method == 'POST':
        name = request.form.get("name")
        email = request.form.get("email")

        resp = make_response(redirect("/hello"))
        resp.set_cookie("name", name)
        resp.set_cookie("email", email)
        return resp

    return render_template("index.html")


@app.route("/hello", methods=['GET',"POST"])
def hello():
    if request.method == 'POST':
        response = make_response(redirect("/"))
        response.delete_cookie("name")
        response.delete_cookie("email")
        return response

    name = request.cookies.get("name")
    email = request.cookies.get("email")
    return render_template("hello.html", name=name, email=email)
```